tpl/tplimpl/embedded/templates/_default/sitemap.xml

```diff
@@ -1,7 +1,7 @@
 {{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
   xmlns:xhtml="http://www.w3.org/1999/xhtml">
-  {{ range .Data.Pages }}
+  {{ range where .Data.Pages "Params.sitemap.exclude" "ne" true }}
```

Comment:
* Add `Exclude` to the `SitemapConfig` struct.
* Change the above where clause to use `.Sitemap.Exclude`
* We also need a test for this (I guess `tplimpl_integration_test.go` would be a OK placement)

Reply:
Make that `.Sitemap.Disable` -- which is in line with what we have elsewhere.